From 66fc030ff88f8e5602e4055cdeb2c6af2fa3e563 Mon Sep 17 00:00:00 2001
From: Kha Tran <kha.tran.px@rvc.renesas.com>
Date: Fri, 19 Jan 2018 18:34:02 +0700
Subject: [PATCH 313/628] ARM: DTS: r8a7744: Add CMT devices

Add the CMT0 and CMT1 counters to the r8a7744 device tree and make them
disabled by default.

Signed-off-by: thongsyho <thong.ho.px@rvc.renesas.com>
Signed-off-by: Kha Tran <kha.tran.px@rvc.renesas.com>
Signed-off-by: vietn <vietn@fsoft.com.vn>
---
 arch/arm/boot/dts/r8a7744.dtsi | 32 ++++++++++++++++++++++++++++++++
 1 file changed, 32 insertions(+)

diff --git a/arch/arm/boot/dts/r8a7744.dtsi b/arch/arm/boot/dts/r8a7744.dtsi
index 175ec3e..78bde73 100644
--- a/arch/arm/boot/dts/r8a7744.dtsi
+++ b/arch/arm/boot/dts/r8a7744.dtsi
@@ -162,6 +162,38 @@
 			     <1 10 (GIC_CPU_MASK_SIMPLE(2) | IRQ_TYPE_LEVEL_LOW)>;
 	};
 
+	cmt0: timer@ffca0000 {
+		compatible = "renesas,cmt-48-r8a7744", "renesas,cmt-48-gen2";
+		reg = <0 0xffca0000 0 0x1004>;
+		interrupts = <0 142 IRQ_TYPE_LEVEL_HIGH>,
+		<0 143 IRQ_TYPE_LEVEL_HIGH>;
+		clocks = <&mstp1_clks R8A7744_CLK_CMT0>;
+		clock-names = "fck";
+
+		renesas,channels-mask = <0x60>;
+
+		status = "disabled";
+	};
+
+	cmt1: timer@e6130000 {
+		compatible = "renesas,cmt-48-r8a7744", "renesas,cmt-48-gen2";
+		reg = <0 0xe6130000 0 0x1004>;
+		interrupts = <0 120 IRQ_TYPE_LEVEL_HIGH>,
+		<0 121 IRQ_TYPE_LEVEL_HIGH>,
+		<0 122 IRQ_TYPE_LEVEL_HIGH>,
+		<0 123 IRQ_TYPE_LEVEL_HIGH>,
+		<0 124 IRQ_TYPE_LEVEL_HIGH>,
+		<0 125 IRQ_TYPE_LEVEL_HIGH>,
+		<0 126 IRQ_TYPE_LEVEL_HIGH>,
+		<0 127 IRQ_TYPE_LEVEL_HIGH>;
+		clocks = <&mstp3_clks R8A7744_CLK_CMT1>;
+		clock-names = "fck";
+
+		renesas,channels-mask = <0xff>;
+
+		status = "disabled";
+	};
+
 	irqc0: interrupt-controller@e61c0000 {
 		compatible = "renesas,irqc-r8a7744", "renesas,irqc";
 		#interrupt-cells = <2>;
-- 
2.7.4

